src/controllers/urls.go

```diff
@@ -88,8 +92,15 @@ func CreateUrlsController() *UrlsController {
 }
 
 func (c *UrlsController) CreateSpecificShortUrl(shortUrl string, longUrl string, userId uint64) (url *models.Url, err error) {
+	id, err := utils.Radix64Decode(shortUrl)
+
+	if err != nil {
+		applogger.Error("CreateSpecificShortUrl: ", err)
```

Comment:
not all errors inside radix function are because of this. 
generate this error only if len(shortUrl) > 10